src/@orb.yml

```
@@ -10,3 +10,6 @@ description: >
 
 orbs:
   continuation: circleci/continuation@0.1.2
+  display:
```

seems like the indentation is off? 

```suggestion

display:
  source_url: "https://github.com/roopakv/orbs"
  home_url: "https://orb.swissknife.dev"
```